packages/gestalt/src/SideNavigation.css

```diff
@@ -14,13 +15,39 @@
 
 .fullHeight {
   height: 100%;
-  overflow: scroll;
+  overflow-y: scroll;
+}
+
+.contentWidthTransition {
+  transition: width 0.2s;
 }
 
 .section {
+  margin-bottom: var(--space-600);
   margin-top: var(--space-600);
 }
 
-.section:nth-child(1) {
+.section:first-child {
   margin-top: 0;
 }
+
+.section:last-child {
+  margin-bottom: 0;
+}
+
+.sectionStartDivider {
+  display: none;
```

Comment:
Section dividers appear when `SideNavigation` is collapsed. 
There are 2 dividers for each section, top (sectionStartDivider) and bottom (sectionEndDivider).

Bottom divider is always displayed, *except* the section is the last child.

Top divider is always hidden. Because if there are 2 sections stacked, in the middle there appears 2 dividers (1st one's bottom divider and 2nd's top divider).

Top divider appears only when `TopItem` or `Group` comes before `Section`.

Reply:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/3815a71e-5b63-4b7a-9a84-4d646d158ba8)